server/mailet/mailets/src/test/java/org/apache/james/transport/mailets/RecipientRewriteTableTest.java

```diff
@@ -18,131 +18,79 @@
  ****************************************************************/
 package org.apache.james.transport.mailets;
 
-import static org.apache.james.transport.mailets.RecipientRewriteTableMock.mapFrom;
-import static org.apache.james.transport.mailets.RecipientRewriteTableMock.rewriteTableMock;
-import static org.junit.Assert.assertEquals;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.when;
 
-import java.util.Iterator;
 import java.util.Properties;
 
-import javax.mail.Address;
-import javax.mail.Message;
-import javax.mail.MessagingException;
 import javax.mail.Session;
-import javax.mail.internet.AddressException;
 import javax.mail.internet.MimeMessage;
 
-import org.apache.mailet.Mail;
-import org.apache.mailet.MailAddress;
+import org.apache.james.domainlist.api.DomainList;
 import org.apache.mailet.MailetContext;
+import org.apache.mailet.base.MailAddressFixture;
 import org.apache.mailet.base.test.FakeMail;
 import org.apache.mailet.base.test.FakeMailContext;
 import org.apache.mailet.base.test.FakeMailetConfig;
-import org.junit.After;
 import org.junit.Before;
 import org.junit.Test;
-
-import com.google.common.base.Function;
-import com.google.common.base.Throwables;
-import com.google.common.collect.FluentIterable;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
 
 public class RecipientRewriteTableTest {
 
-    private static final Session NO_SESSION = null;
-    
-    private org.apache.james.transport.mailets.RecipientRewriteTable table;
+    private RecipientRewriteTable mailet;
+
+    @Mock org.apache.james.rrt.api.RecipientRewriteTable virtualTableStore;
+    @Mock DomainList domainList;
+    @Mock MailetContext mockMailetContext;
```

Comment:
In some cases we just need to verify the MailetContext's methods are called (like logging, sendMail) so I think we still keep the mock.